modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -27,75 +28,191 @@ UnitTest.asynctest('browser.tinymce.core.content.EditorContentTest', (success, f
     return htmlSerializer.serialize(node);
   };
 
+  let events = [];
+  const logEvent = (event: EditorEvent<{}>) => {
+    events.push(event.type);
+  };
+
+  const sAssertEventsFiredInOrder = (expectedEvents?: string[]) => Step.sync(() => {
+    Assertions.assertEq('Get content events should have been fired', expectedEvents || [
+      'beforesetcontent',
+      'setcontent',
+      'beforegetcontent',
+      'getcontent'
+    ], events);
+  });
+  const sClearEvents = () => Step.sync(() => events = []);
```

Comment:
```suggestion
  const sClearEvents = Step.sync(() => events = []);
```
Consider unthunking this?